ghost/member-events/lib/PaidSubscriptionStartedEvent.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Had 2 minds about overloading the existing SubscriptionCreatedEvent vs adding a new one which captures all the information for paid susbcription, and a new one seemed better as the two have different usecases.

Reply:
I think using the `SubscriptionCreatedEvent` makes more sense here ☺️ We have an issue to move it here: https://github.com/TryGhost/Team/issues/1881

So would be great if you could use that instead

Reply:
Ah! So we'd want the `SubscriptionCreatedEvent` to be moved to where we have the new `PaidSubscriptionStartedEvent` in `linkSubscription` right for that issue? Guess that'll also need to pass down the attribution changes ? And how do you feel about overloading it with full data instead of IDs ?

Reply:
Resolved below, gonna switch to using ids